irohad/torii/processor/impl/transaction_processor_impl.cpp

```diff
@@ -95,16 +95,15 @@ namespace iroha {
     }
 
     void TransactionProcessorImpl::transactionHandle(
-        std::shared_ptr<model::Transaction> transaction) {
+        std::shared_ptr<shared_model::interface::Transaction> transaction) {
       log_->info("handle transaction");
       model::TransactionResponse response;
```

Comment:
Is it possible to replace it by shared_model::interface::TransactionResponse at this point?